d2l-table.js

```diff
@@ -161,7 +161,7 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-table">
 				};
 			}
 		</style>
-		<d2l-scroll-wrapper show-actions="">
+		<d2l-scroll-wrapper show-actions="" needs-table>
```

Comment:
Could we simplify and just always do `<d2l-scroll-wrapper role="table">` here, dropping the `needs-table` attribute entirely?

Reply:
Unfortunately no. If `<d2l-table-wrapper><table>...` is used, the double table role/element also can confuse the screenreaders.

Reply:
Lovely. And I guess when `d2l-scroll-wrapper` is used without `<d2l-table>`, then everything is good, hence why this is configurable.

Reply:
Precisely.